rfc081-release-cadence.md

```diff
@@ -41,6 +41,76 @@ potential hot fixes or follow-up.
 Major releases in April avoids releasing during winter holidays, summer
 vacations, ChefConf and Chef Summits.
 
+## Cookbook and Ecosystem Tooling Support
+
+The latest version of Chef-managed community cookbooks should support at least the latest 6
+months of Chef Client versions.  After 6 months, Chef-managed community cookbooks may
+elect to drop support for older Chef Client versions.  This window does not reset on
+a major version release so that the prior major version track is supported for a 6 month
+window.  Non-Chef-managed community cookbooks are encouraged to follow this policy.
+
+As an example, in May we will typically release 14.1.0 and both 14.0 and 14.1 will be
+supported.  The version of 13 release 6 months prior should be 13.8 and will still
+be supported so that 6 versions will be considered current (13.8 through 13.11 plus
+14.0 and 14.1).  At that point community cookbooks may choose to start using 13.8
+features and drop support for versions prior to 13.7
+
+Tooling external to cookbooks (cookstyle, chefspec, stove, foodcritic, halite,
+poise-hoist, etc) is similarly encouraged to follow this policy.
+
+## Rationale
+
+This allows cookbook authors to drop support for old cookbook versions and use new
+features in new client versions.  It gets us out of the situation where cookbooks still
+need to support 28-month old versions of Chef Client versions and can't use any features
+added in the past 28-months of software development.
+
+## Ruby Cadence
+
+Since the Ruby language itself releases new minor versions over the Christmas holidays,
+the April major release of Chef Client should include the minor revision of Ruby which
+landed the prior Christmas.  Combined with the 6 month sliding window for cookbook
+support that also implies that when the prior major release of the client falls off
+of community cookbook support that the prior minor release of Ruby will also fall
+off of community cookbook support (including the cookstyle gem and related tooling).
+
+The release of the new major version may be delayed if there are show stopping bugs
+in the released version of Ruby (we assume that 4 months will be enough time for
+major regressions in the core language to be addressed, but that is an external
+dependency).
+
+If the Ruby language version released over Christmas has a show-stopper bug then the
+next major Chef Client version may be released without it.  It can then be included
+in a subsequent minor version bump.  This RFC deliberately uses 'should' instead of
+'must', and [RFC-034](https://github.com/chef/chef-rfc/blob/b7bd9c53bf96235f9334e65bb5848f7843c81fed/rfc034-ruby-193-eol.md#specification)
```

Comment:
i actually did that deliberately so that if we edited that RFC to say something different that we'd follow the link to what the text said when this was written.

i can see the point of referencing the current RFC, but if that RFC was updated with incompatible language, then with a link that tracked the current version it just becomes deeply confusing if you follow the link.  if you follow the link to an old RFC it becomes more apparent what happened.